### PR TITLE
DG-76, DG-16 Modifying Wildlife Spotter transcriptions and auto-validation threshold

### PR DESCRIPTION
**Pull request by cdausmus**
- Implemented template-level setting to ignore animal count
- Updated UX for WS project transcriptions
-- Hide animal count when not required (from template setting)
-- Initialise count at zero
-- Use Bootstrap number field with built-in +/- controls
-- Disable save button until all selected animals count > 0
- Cleaned up unnecessary/commented out code

### CODE REVIEW

File: grails-app/assets/javascripts/transcribe/wildlifespotter.js [hunk 1]
```diff
@@ -110,14 +133,20 @@ function wildlifespotter(wsParams, imagePrefix, recordValues, placeholders) {
       generateFormFields();
     });
 
-    $('#ct-container').on('change', 'select.numAnimals', function() {
+    //$('#ct-container').on('change', 'select.numAnimals', function() {
```
Comment (chrisala) [inline grails-app/assets/javascripts/transcribe/wildlifespotter.js:136]: Not sure about the style guidelines you normally follow but I'd tend to delete commented code like this.

File: grails-app/assets/javascripts/transcribe/wildlifespotter.js [hunk 2]
```diff
@@ -110,14 +133,20 @@ function wildlifespotter(wsParams, imagePrefix, recordValues, placeholders) {
...
+    $('#ct-container').on('change', 'input.numAnimals', function() {
       var $this = $(this);
       var idx = $this.closest('[data-item-index]').data('item-index');
       var count = $this.val();
+      // console.log("value change: " + count);
```
Comment (chrisala) [inline grails-app/assets/javascripts/transcribe/wildlifespotter.js:141]: Consider deleting this one.

**Review by chrisala — APPROVED** [review]
Consider deleting commented console.logs.
This is more of a question, but when you have the "Validate on species only" it's excluding individual count from the form and validation, but submitting an implicit "1" for the darwin core individual count field.    Is this the intended behaviour? (I'm assuming it's not a major issue for the data if the image had two individuals but only one is recorded)

File: grails-app/assets/javascripts/transcribe/wildlifespotter.js (hunk 1 above)
Comment (cdausmus) [reply grails-app/assets/javascripts/transcribe/wildlifespotter.js:136]: I usually do too. Must have missed it. Thanks.

File: grails-app/assets/javascripts/transcribe/wildlifespotter.js (hunk 2 above)
Comment (cdausmus) [reply grails-app/assets/javascripts/transcribe/wildlifespotter.js:141]: Thanks.

**Comment by cdausmus** [comment]
> Consider deleting commented console.logs. This is more of a question, but when you have the "Validate on species only" it's excluding individual count from the form and validation, but submitting an implicit "1" for the darwin core individual count field. Is this the intended behaviour? (I'm assuming it's not a major issue for the data if the image had two individuals but only one is recorded)

It was intentional, however, a good point. I don't believe it will be an issue but I'll double check anyway. Thanks for the input.

**Pull request merged**